packages/wasm/crate/src/planner.rs

```diff
@@ -205,7 +205,7 @@ pub async fn plan_transaction_inner<Db: Database>(
     let gas_prices = storage
         .get_gas_prices_by_asset_id(&fee_asset_id)
         .await?
-        .ok_or_else(|| anyhow!("GasPrices not available"))?;
+        .unwrap_or_default();
```

Comment:
see comment below addressing https://github.com/penumbra-zone/web/issues/1602

Reply:
I think it's a mistake, we shouldn't use default gas prices
And use only those gas prices that have been published in compact blocks

Reply:
I agree with this sentiment. For instance, this would allow paying for fees with non-supported gas tokens, for instance `TestUSD`.